_docs/setup/kubernetes/quick-start.md

```diff
@@ -87,6 +87,11 @@ run the following command to download and extract the latest release automatical
     * The `istioctl` client binary in the `bin/` directory. `istioctl` is used when manually injecting Envoy as a sidecar proxy and for creating routing rules and policies.
     * The `istio.VERSION` configuration file
 
+1. Change directory to istio package. For example, if the package is istio-0.2.7
+  ```bash
+  cd istio-0.2.7
+  ```
```

Comment:
Use {data.version} (I can check the exact syntax from my phone sorry, need to check the footer for an example) or mention that is just an example and the version could be different

Reply:
I mentioned 'For example, if the package is istio-0.2.7...'

Reply:
sorry quick reading on my phone, I missed that - thx (still could use the version variable but it's fine)